utils/node_local_test.py

```diff
@@ -3999,13 +3960,13 @@ def test_alloc_fail_copy(server, conf, wf):
     pool = server.get_test_pool()
 
     src_dir = tempfile.TemporaryDirectory(prefix='copy_src_',)
-    ofd = open(os.path.join(src_dir.name, 'file'), 'w')
+    ofd = open(join(src_dir.name, 'file'), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations